tests/components/dynalite/test_config_flow.py

```diff
@@ -104,3 +106,71 @@ async def test_two_entries(hass):
         )
     assert result["type"] == "create_entry"
     assert result["result"].state == "loaded"
+
+
+async def test_user_flow(hass):
+    """Run a flow generated by the user."""
+    host = "1.2.3.4"
+    with patch(
+        "homeassistant.components.dynalite.bridge.DynaliteDevices.async_setup",
+        return_value=True,
+    ):
+        result = await hass.config_entries.flow.async_init(
+            dynalite.DOMAIN,
+            context={"source": config_entries.SOURCE_USER},
+        )
+        assert result["type"] == "form"
+        assert result["step_id"] == "user"
+        result = await hass.config_entries.flow.async_configure(
```

Comment:
We want to patch the integration `async_setup_entry` on create entry results to avoid setting up the whole integration.

Reply:
done